packages/sdk/scripts/rewrite-package.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * This script moves the package.json file into the dist folder while adjusting
+ * relative pathnames for exports. Since the compiled output is inside "dist",
+ * but the original package.json references paths as if it's in the root, we
+ * need to strip "dist" from all export fields (main, types, …).
+ *
+ * This ensures that consumers of the package import the correct files
+ * without referencing "dist" in their paths, maintaining proper module resolution.
+ */
+
+import pkg from "../package.json" assert { type: "json" }
```

Comment:
Would it make sense to unify the two rewrite-package.ts scripts and move that unified script to somewhere where both packages can access it? E.g. a new `scripts` directory in monorepo root.

Reply:
I wouldn't press on it at this point. 2 scripts are totally manageable atm.

This could be a separate task: identify more shared scripts and have a `scripts` package then.

I'll leave it be for now.